build.sbt

```diff
@@ -1016,6 +1016,7 @@ lazy val `dev-environment` = (project in file("dev"))
     publish := {},
     PgpKeys.publishSigned := {}
   )
+  .disablePlugins(Unidoc)
```

Comment:
The problem here is:

1. The `Unidoc` plugin (defined in [`project/Doc.scala`](https://github.com/lagom/lagom/blob/91d875989924176d457a5fc5ea4dc8c0f094a984/project/Doc.scala#L153-L187)) adds a dependency on `genjavadoc`
2. `genjavadoc` is cross-built for _every_ Scala version, not just majors
3. Only `genjavadoc` 0.11 has been [built for Scala 2.12.6](https://github.com/lightbend/genjavadoc/issues/138), so this requires updating to 0.11
4. 0.11 has also [dropped support for Scala 2.10](https://github.com/lightbend/genjavadoc/issues/118)

So I had to disable the `Unidoc` plugin for all of the projects that cross-build to 2.10 (dependencies of the sbt plugin basically, and also ones that don't define a `scalaVersion` now that I look more closely).

This works, and I think it's OK because none of these projects really have API docs in the first place, but I'm not sure if it's the best way to handle this. I'm not that knowledgable about sbt plugin development yet, and I don't fully understand everything happening in `Doc.scala`.

For example, there's something that looks like it should filter this down to just Java DSL and Scala DSL projects:

https://github.com/lagom/lagom/blob/91d875989924176d457a5fc5ea4dc8c0f094a984/project/Doc.scala#L51-L52

Evidently, that either doesn't work, or I'm misunderstanding what it's meant to do.